spark/src/main/java/org/opensearch/sql/spark/dispatcher/BatchQueryHandler.java

```diff
@@ -28,6 +28,10 @@
 import org.opensearch.sql.spark.leasemanager.model.LeaseRequest;
 import org.opensearch.sql.spark.response.JobExecutionResponseReader;
 
+/**
+ * The handler for batch query. With batch query, queries are executed as single batch. The queries
+ * are sent along with job execution request to spark.
```

Comment:
what is job execution request? can you add a ref link to the class?

Reply:
Fixed.